client/chain/ofac.go

```diff
@@ -40,11 +38,7 @@ func NewOfacChecker() (*OfacChecker, error) {
 }
 
 func getOfacListPath() string {
-	currentDirectory, _ := os.Getwd()
-	for !strings.HasSuffix(currentDirectory, "sdk-go") {
-		currentDirectory = filepath.Dir(currentDirectory)
-	}
-	return filepath.Join(filepath.Join(filepath.Join(currentDirectory, "client"), "metadata"), ofacListFilename)
+	return getFileAbsPath(fmt.Sprintf("../metadata/%s", ofacListFilename))
```

Comment:
We should not make assumptions regarding the directory separator used by the OS. To create a path we should be using `path.Join`